### DISCUSSION TITLE
Sub-module for Google Cloud Platform

### DISCUSSION THREAD

**Discussion by AlbertMoser** [dropwizard/metrics#2830] [Ideas]
Hi Dropwizard team,
I wanted to ask, if there are any plans to create a sub-module similar to the Graphite sub-module, for reporting metrics to Google Cloud Platform via their monitoring API (https://cloud.google.com/monitoring/custom-metrics/creating-metrics). We would like to make use of the metrics reporter in our cloud infrastructure.
Thank you very much.

**Comment by joschi** [comment]
Unless you or some other member of the community wants to work on this and maintain it over longer time, there are no plans to adopt any additional _official_ modules for now.

This being said, I think these kind of integrations are best as a third-party module, ideally somehow supported by the actual vendor (in this case Google Cloud).

**Comment by AlbertMoser** [comment]
I would have to check with my team in more detail if someone would like to work on it and maintain it, since I'm not that well versed in Java.
